### ISSUE TITLE
Standardize on s-status-editing

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmctweb#468]
Primarily in /wtd-dev/platform/commonUI/browse/res/templates/browse-object.html and /wtd-dev/platform/commonUI/browse/res/templates/browse.html. Note that location of s-status-editing when applied may be different in the markup - it'll be on the representation.

Do this as a branch off of `nem_prototype`.

**Comment by charlesh88** [comment]
@akhenry Added this issue, will get to asap.

**Comment by charlesh88** [comment]
In browse-object.html, get rid of ng-class statement regarding .edit-mode; this is being superceded by s-status-editing which is being added on enclosing representations.

**Comment by charlesh88** [comment]
@akhenry Starting work on this.

**Comment by charlesh88** [comment]
@akhenry I've refactored the main edit area and the Inspector to now use s-status-editing, and removed the relevant ng-class and ng-show statements. I believe I've hit all instances related to NEM that we wanted to refactor. Work has been pushed to branch nem468. 

One note: it'd be really nice if the Elements pool in the Inspector was able to retain its user-set height as well as transition in and out when going from browse to edit mode, and I added CSS to allow this. However, it's not working currently, as it appears that the entire Inspector is completely re-created when going from object to object, or when going from browse to edit mode. If there's a way to make this more efficient and allow us to retain user settings within the Inspector, that would be desired. Something to think about as we move from prototype to real architecture for NEM?
